addons/binding/org.openhab.binding.airquality/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,164 @@
+    
+    <channel-type id="o3">
+        <item-type>Number</item-type>
+        <label>O3</label>
+        <description>Ozone level</description>
+        <category>O3</category>
+        <state readOnly="true" pattern="%d"></state>
+    </channel-type>
+
+    <channel-type id="no2">
+        <item-type>Number</item-type>
+        <label>NO2</label>
+        <description>Nitrogen dioxide level</description>
+        <category>NO2</category>
+        <state readOnly="true" pattern="%d"></state>
+    </channel-type>
+
+    <channel-type id="co">
+        <item-type>Number</item-type>
+        <label>CO</label>
+        <description>Carbon monoxide level</description>
+        <category>CO</category>
+        <state readOnly="true" pattern="%d"></state>
+    </channel-type>
+
+    <channel-type id="locationName" advanced="true">
+        <item-type>String</item-type>
+        <label>Location</label>
+        <description>Nearest measuring station location</description>
+        <category>Location</category>
+        <state readOnly="true" pattern="%s"></state>
+    </channel-type>
+
+    <channel-type id="locationUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>URL</label>
+        <description>Direct URL to the station results</description>
+        <category>URL</category>
+        <state readOnly="true" pattern="%s"></state>
+    </channel-type>
+
+    <channel-type id="observationTime" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Observation time</label>
+        <description>Observation date and time</description>
+        <category>Observation time</category>
+        <state readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="stationId" advanced="true">
+        <item-type>Number</item-type>
+        <label>Station ID</label>
+        <description>Measurement station identifier</description>
+        <category>Station ID</category>
+        <state readOnly="true" pattern="%d"></state>
+    </channel-type>  
+
+    <channel-type id="temperature" advanced="true">
+        <item-type>Number</item-type>
+        <label>Temperature (°C)</label>
+        <description>Temperature in Celsius degrees</description>
+        <category>Temperature (°C)</category>
+        <state readOnly="true" pattern="%.1f °C"></state>
+    </channel-type>
+
+    <channel-type id="pressure" advanced="true">
+        <item-type>Number</item-type>
+        <label>Pressure level</label>
+        <description></description>
+        <category>Pressure level</category>
+        <state readOnly="true" pattern="%.1f hPa"></state>
+    </channel-type>
+
+    <channel-type id="humidity" advanced="true">
+        <item-type>Number</item-type>
+        <label>Humidity level</label>
+        <description></description>
+        <category>Humidity level</category>
+        <state readOnly="true" min="0" max="100" pattern="%d %%">
+        </state>
+    </channel-type>
+
+</thing:thing-descriptions>
```

Comment:
Could you add a line feed at the end of the file? Makes things nicer to read on the shell.